app/jobs/spotlight_search/export_job.rb

```diff
@@ -0,0 +1,33 @@
+module SpotlightSearch
+  class ExportJob < ApplicationJob
+    def perform(email, records, columns = nil, filters = nil)
+      # data = get_data(records.class, filters)
+      file_path = create_excel(records.as_json, records.class.name)
+      subject = "#{records.class.name&.humanize} export at #{Time.now}"
+      ExportMailer.send_excel_file(email, file_path, subject).deliver_now
```

Comment:
```suggestion
      ExportMailer.send_excel_file(email, file_path, subject).deliver_later
```

Reply:
since this is already a background job and the only process to be executed in the thread, why again push a mail to another background job